### PR TITLE
fix: Use setup future usage for saving card

### PR DESCRIPTION
**Pull request by manoj-juspay**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
In case its not a mandate transaction , check for `setup_future_usage` flag to save a card

### Additional Changes

- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code

### CODE REVIEW

File: crates/api_models/src/enums.rs [hunk 1]
```diff
@@ -552,6 +552,7 @@ pub enum RoutableConnectors {
     Globalpay,
     Klarna,
     Payu,
+    Rapyd,
```
Comment (Abhicodes-crypto) [inline crates/api_models/src/enums.rs:555]: How is this change related to this fix ?

Comment (manoj-juspay) [reply crates/api_models/src/enums.rs:555]: its not just saw Rapyd missing

**Review by jarnura — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]
